packages/hooks/src/useTradingPlatformStatus.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from '@deriv/api';
+import useAuthorize from './useAuthorize';
+
+/** A custom hook that gets the list of statuses of ctrader dxtrade mt5 platform. */
+const useTradingPlatformStatus = () => {
+    const { isSuccess } = useAuthorize();
+    const { data, ...rest } = useQuery('trading_platform_status', {
+        options: { enabled: isSuccess },
+    });
+
+    const tradingPlatformStatusData = data?.trading_platform_status;
```

Comment:
I think it would be nice if we can return specific platforms here instead of giving the main response. What if we need to add some stuff to specific platforms in the future.